### DISCUSSION TITLE
中文交流群

### DISCUSSION THREAD

**Discussion by zuohuadong** [nestcn/docs.nestjs.cn#250] [1-公告]
### QQ 群

一群：  [277386223](https://jq.qq.com/?_wv=1027&k=aaetFYo1)

二群： [1031015552](https://jq.qq.com/?_wv=1027&k=Ay5jLfqo)

### 微信交流群： 

一群：（已满）    

【内部邀请】

二群：    

<img src="https://pic.downk.cc/item/5f02e2e114195aa594ed9f59.jpg" alt="加微信" width="260" height="260" align="bottom" />

### 关注

[左华栋的知乎](https://www.zhihu.com/people/dongcang)    
[汪志成的知乎](https://www.zhihu.com/people/alpha-gde)    
[justjavac的知乎](https://www.zhihu.com/people/justjavac.com)

### 互联网交流（闲聊）    

左华栋的互联网圈：    

<img src="https://pic.downk.cc/item/5f8c6c341cd1bbb86b72fc9a.jpg" alt="加微信" width="260" height="260" align="bottom" />

**Comment by Mahoo12138** [comment]
希望增加 QQ 频道
